### ISSUE TITLE
Hero + InputAccessoryView Bug

### ISSUE THREAD

**Issue by OrAzr** [HeroTransitions/Hero#569]
## What did you do?

I have override inputAccessoryView of UIViewController (the bottom yellow view) and noticed a bug thats happening when i pop view controller where the keyboard is presented, when i pop back to viewcontroller that show inputAccessoryView, The inputAccessoryView suddenly disconnected from where it supposed to be.
the most strange part is the view debbuger shows the inputAccessoryView at the correct place.  
i have fiddled with it 4 days now :(

## What did you expect to happen?

i expected the InputAccessoryView to always have constraint to the bottom of the superview, like it always did.

##  What happened instead?

the InputAccessoryView is detached from the superview bottom

## General Information

1.3.1

12.*

3,4

* Devices/Simulators:

* Reproducible in Examples? yes

## Demo Project

https://www.youtube.com/watch?v=T76wgl1qUso

link to demo project:
https://drive.google.com/file/d/145l1j30Bib_qzEj7Nh8KPkNPnNVD9FLd/view?usp=sharing

**Comment by ngoccuong291** [comment]
I got the same issue, took me a few days already, at first I thought there was something wrong with my code but I can replicate the same issue with the example code we have in this repo. @lkzhao do we have any light on this issue? If there is some hacks that we can apply to fix it quickly, I'm more than happy to give it a shot, thanks. I made a fork: https://github.com/ngoccuong291/Hero.git the testing code is sitting on branch "defect/InputAccessoryViewIssue" 
![Simulator Screen Shot - iPhone Xʀ - 2019-07-09 at 14 58 56](https://user-images.githubusercontent.com/1083098/60860682-f7396180-a25a-11e9-81e1-641f4b5a5c95.png)
